homeassistant/components/flux_led/light.py

```diff
@@ -210,6 +210,7 @@ def brightness(self):
             return self._white_value
 
         return int(self._color[2] / 100 * 255)
+        return int(self._color[2] / 100 * 255) or self._white_value
```

Comment:
Since, now we are working in both RGB and "warm white" mode where we are storing RGB in HSV format (`self._color`). Brightness can be represented either as HSV[2] *or* white value